router/handlers.go

```diff
@@ -0,0 +1,30 @@
+package router
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func (rt *Router) handleHome(w http.ResponseWriter, r *http.Request) {
+	_, err := fmt.Fprint(w, "Welcome to Budget Helper!\n")
+	if err != nil {
+		log.Fatalf("handleHome: %v\n", err)
+	}
+}
+
+func (rt *Router) handleUsers(w http.ResponseWriter, r *http.Request) {
+	_, err := fmt.Fprint(w, "Fetching users...\n")
+	if err != nil {
+		log.Fatalf("handleUsers: %v\n", err)
+	}
+
+	userList, err := rt.UserRepo.GetAllUsers()
+	if err != nil {
+		log.Fatalf("GetAllUsers: %v\n", err)
```

Comment:
```
handleUsers: %v\n
```

The error will suggest which function that blew up (eg. `user not found`), we want to know where it was called.